### PR TITLE
Die when validate_script_output called without $code

### PR DESCRIPTION
**Pull request by asmorodskyi**
No sense of using validate_script_output without callback code
for validation so function will fail with
proper error message when it is happening

### CODE REVIEW

File: testapi.pm [hunk 1]
```diff
@@ -907,12 +907,12 @@ Wrapper around script_output, that runs a callback on the output. Use it as
 
 =cut
 
-sub validate_script_output($&;$) {
+sub validate_script_output {
```
Comment (okurz) [inline testapi.pm]: good but unrelated. Please put into at least another commit if not PR where you try to remove the signature for all sub's where possible

Comment (coolo) [inline testapi.pm]: Not good actually - did you test this?

Comment (coolo) [inline testapi.pm]: So I think I want a test case

**Comment by asmorodskyi** [comment]
it's good that @coolo suggested to add tests :) 
they show that with ($;$) you actually can't reach die message , so revert back ($;$) but still you can't reach "return unless $code" too , so this commit is about removing one useless line and add 100% test coverage for the method :)

**Comment by okurz** [comment]
LGTM

**Comment by coolo** [comment]
I knew your change wouldn't work - so thanks for the test case :)
